contracts/VaultV3.vy

```diff
@@ -200,15 +200,14 @@ enum Rounding:
     ROUND_DOWN
     ROUND_UP
 
-# IMMUTABLE #
+# STORAGE #
 # Underlying token used by the vault.
-ASSET: immutable(ERC20)
```

Comment:
kind of sad losing immutables

Reply:
Agreed.

We will have to do some more research on the clones with immutables to see if it can be done in vyper